ci/gpu/build.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
```

Comment:
This file is for GPU testing:
1. Build from source
2. Run tests
3. Capture test output from pytest to report stats